concept/type_/annotation.rs

```diff
@@ -79,7 +81,12 @@ impl AnnotationCardinality {
         Self { start_inclusive, end_inclusive }
     }
 
+    // TODO: Can move to impl Default for AnnotationCardinality and other Annotations
     pub const fn default() -> Self {
+        Self::unchecked()
+    }
+
+    pub const fn unchecked() -> Self {
```

Comment:
Why is this called `unchecked`?

Reply:
The idea was like "this card is so permissive that we don't really need to check it", can be used for optimizations (e.g. when we create locks)